src/crypto/DeviceList.js

```diff
@@ -415,9 +555,8 @@ export default class DeviceList {
  * time (and queuing other requests up).
  */
 class DeviceListUpdateSerialiser {
-    constructor(baseApis, sessionStore, olmDevice) {
+    constructor(baseApis, olmDevice) {
```

Comment:
As above this now returns updated device lists for the given users, rather than interacting with the store directly.